telemetry/sql/000005_pushed_envelope.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE publish_method AS ENUM ('LightPush', 'Relay');
+
+CREATE TABLE IF NOT EXISTS sentEnvelopes (
+    id SERIAL PRIMARY KEY,
+    messageHash VARCHAR(255) NOT NULL,
+    sentAt INTEGER NOT NULL,
+    createdAt INTEGER NOT NULL,
+    topic VARCHAR(255) NOT NULL,
+    pubSubTopic VARCHAR(255) NOT NULL,
+    senderKeyUID VARCHAR(255) NOT NULL,
+    nodeName VARCHAR(255) NOT NULL,
+    processingError VARCHAR(255),
```

Comment:
I think this field for sent envelopes is not needed.
They are needed for received envelopes because they are processed by the app.

Reply:
Removed in latest commit